pyproject.toml

```diff
@@ -53,3 +53,4 @@ pytest-asyncio = "^0.23.5"
 [tool.poetry.group.dev.dependencies]
 ipdb = "^0.13.13"
 mypy = "^1.10.0"
+ipykernel = "^6.29.4"
```

Comment:
what's this for?

Reply:
This is to integrate [jupyter notebooks](https://jupyter.org/) into vscode. I  use them a lot for POCs and quick ideas